.github/workflows/main.yml

```diff
@@ -50,6 +50,11 @@ jobs:
         rustup update $TOOLCHAIN --no-self-update && rustup default $TOOLCHAIN
       shell: bash
       if: matrix.build == 'msrv'
+    - name: Run cargo doc, deny warnings
+      run: |
+        RUSTDOCFLAGS="-D warnings" cargo doc -p bzip2-sys --no-deps
+        RUSTDOCFLAGS="-D warnings" cargo doc -p bzip2 --no-deps
+        RUSTDOCFLAGS="-D warnings" cargo doc -p bzip2 --no-deps --no-default-features --features libbz2-rs-sys
```

Comment:
You can also use the `env` key like this: https://github.com/trifectatechfoundation/sudo-rs/blob/2b0d8a1e3b999ea3c5fb9435feea70508a449c9d/.github/workflows/ci.yaml#L136-L137

Reply:
Also we aren't denying warnings for build jobs either.

Reply:
we are for clippy though, so that would report any issues right?

Reply:
Right